src/utils/dbg.js

```diff
@@ -2,18 +2,37 @@ import { bindActionCreators } from "redux";
 import * as timings from "./timings";
 import { prefs, features } from "./prefs";
 
-export function setupHelper(obj) {
-  const selectors = Object.keys(obj.selectors).reduce((bound, selector) => {
+function findSource(dbg, url) {
+  const sources = dbg.selectors.getSources();
+  const source = sources.find(s => (s.get("url") || "").includes(url));
+
+  if (!source) {
+    return;
+  }
+
+  return source.toJS();
+}
+
+function sendPacket(dbg, packet, cbk) {
+  dbg.connection.tabConnection.debuggerClient
+    .request(packet)
+    .then(cbk || console.log);
```

Comment:
do we want to console.log if there is no callback? perhaps a better pattern would be to have a default empty function ?